include/types/HttpReply.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef HTTPREPLY_HPP
+#define HTTPREPLY_HPP
+
+#include <string>
+#include <unordered_map>
+
+namespace api
+{
+    /**
+     * @struct HTTPReply
+     * @brief This structure defines an example type for HTTP(S) responses.
+     */
+    struct HTTPReply
+    {
+        /**
+         * @brief This field refers to the response code.
```

Comment:
@brief Refers to [...]

Reply:
I'd go with just "The response code" since it would be weird to have "Refers" right before.